doc/nrf/ug_thingy53.rst

```diff
@@ -302,53 +279,151 @@ See `Updating Nordic Thingy:53 firmware`_ for the detailed procedures on how to
 Thingy:53 application guide
 ***************************
 
-Thingy:53 does not have a built-in J-Link debug IC.
-Because of that, the samples and applications that are compatible with Thingy:53 by default use MCUboot bootloader with serial recovery support.
-Thingy:53-compatible applications can be updated using DFU functionality with either `nRF Connect Programmer`_ or nRF Programmer mobile application.
+The Nordic Thingy:53 does not have a built-in J-Link debug IC.
+Because of that, the samples and applications that are compatible with the Nordic Thingy:53 by default use MCUboot bootloader with serial recovery support.
```

Comment:
Suggest rewording this sentence to: _Because of that, the samples and applications that are compatible with the Nordic Thingy:53 by default include the MCUboot bootloader with serial recovery support._

Include is in my opinion more descriptive as the MCUbootloader is a a separate image that is merged with the application.

Reply:
Applied.